### PR TITLE
Business Name component issue #641

### PR DESCRIPTION
**Pull request by bryan-gilbert**

### CODE REVIEW

File: ppr-ui/src/components/business-model.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+/**
+ * The model for a business, such as a registering party, secured party, debtor
+ */
+export class BusinessModel {
+  private _businessName: string
+
+  /**
+   * Creates a new Business model instance.
+   *
+   * @param businessName the last name of the person.
```
Comment (WalterMoar) [inline ppr-ui/src/components/business-model.ts]: This needs to be updated.

Comment (bryan-gilbert) [reply ppr-ui/src/components/business-model.ts]: done

**Review by WalterMoar — CHANGES_REQUESTED** [review]
One comment still references the person name.

File: ppr-ui/src/components/BusinessName.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    value: {
+      required: true,
+      type: BusinessModel
+    }
+  },
+  setup(_, { emit }) {
+
+    // Callback function for emitting model changes made to the business name.
+    function updateName(newBusinesName: string): void {
+      emit('input', new BusinessModel(newBusinesName))
```
Comment (WalterMoar) [inline ppr-ui/src/components/BusinessName.vue]: Typo in "newBusinesName".

**Review by WalterMoar — APPROVED** [review]

**Pull request merged**